homeassistant/components/weatherflow/sensor.py

```diff
@@ -71,7 +71,8 @@ class WeatherFlowSensorEntityDescription(
 
     def get_native_value(self, device: WeatherFlowDevice) -> datetime | StateType:
         """Return the parsed sensor value."""
-        raw_sensor_data = getattr(device, self.key)
+        if (raw_sensor_data := getattr(device, self.key)) is None:
+            return None
```

Comment:
If its `None` we should probably set the `available` `@property` to `False` instead

Reply:
I've added the available property which will return False if this value is None.